js_modules/dagit/packages/core/src/app/Util.tsx

```diff
@@ -74,14 +74,6 @@ export const formatElapsedTime = (msec: number) => {
   return `${hours}:${twoDigit(min)}:${twoDigit(sec)}`;
 };
 
-export function tokenForAssetKey(key: {path: string[]}) {
```

Comment:
These two functions had to be moved to avoid importing all the downstream imports in Util :-/